tests/test_client_functional.py

```diff
@@ -1231,6 +1231,21 @@ async def handler(request):
     resp.close()
 
 
+async def test_GET_DEFLATE(aiohttp_client: Any) -> None:
```

Comment:
We can mark this as expected to fail per https://pganssle-talks.github.io/xfail-lightning/, then merge it (provided other requests are addressed), and work on the fix separately:
```suggestion
@pytest.mark.xfail(reason="https://github.com/aio-libs/aiohttp/pull/5478")
async def test_GET_DEFLATE(aiohttp_client: Any) -> None:
```

Reply:
it's fails only with picohttpparser, with python implementation it's works expected

Reply:
Then this should be reflected in the PR description.